airbyte-commons-worker/src/main/java/io/airbyte/workers/normalization/NormalizationRunnerFactory.java

```diff
@@ -38,12 +41,22 @@ public class NormalizationRunnerFactory {
 
   public static NormalizationRunner create(final String connectorImageName,
                                            final ProcessFactory processFactory,
-                                           final String normalizationVersion) {
+                                           final String normalizationVersion,
+                                           final String normalizationImage) {
     final var valuePair = getNormalizationInfoForConnector(connectorImageName);
+    final String factoryNormalizationImage = String.format("%s:%s", valuePair.getLeft(), normalizationVersion);
+    log.error("destination normalization runner factory -> {}", factoryNormalizationImage);
+    if (Objects.nonNull(normalizationImage)
+        && !normalizationImage.equalsIgnoreCase(factoryNormalizationImage)) {
+      log.error(
+          "The normalization image name or tag in the definition file is different from the normalization image or tag in the NormalizationRunnerFactory!");
+      log.error(
+          "the definition file value - {}, the NormalizationRunnerFactory value - {}", normalizationImage, factoryNormalizationImage);
+    }
```

Comment:
is there any action to take if we get these logs?

Reply:
currently - no, just to see that different versions are used. In general, this is necessary for testing and making sure that we are getting the correct data from the actor_definition table